src/rust/ensogl/lib/core/src/gui/cursor.rs

```diff
@@ -367,8 +367,8 @@ impl Cursor {
         radius.set_target_value(DEFAULT_RADIUS);
         size.set_target_value(DEFAULT_SIZE());
 
-        let fade_out_spring = inactive_fade.spring() * 0.2;
-        let fade_in_spring  = inactive_fade.spring();
+        let fade_out_spring = inactive_fade.simulator.spring() * 0.2;
+        let fade_in_spring  = inactive_fade.simulator.spring();
```

Comment:
forcing user to know that "animation" uses "simulator" under the hood is leaking internal design to the api. its bad